buildSrc/src/main/kotlin/me/vkryl/plugin/CMakePlugin.kt

```diff
@@ -16,71 +16,68 @@ import Config
 import com.android.build.gradle.BaseExtension
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.kotlin.dsl.getByType
 
 open class CMakePlugin : Plugin<Project> {
-  override fun apply (project: Project) {
-    val androidExt = project.extensions.getByName("android")
-
-    if (androidExt is BaseExtension) {
-      androidExt.apply {
-        externalNativeBuild {
-          cmake {
-            path("jni/CMakeLists.txt")
-          }
+  override fun apply(project: Project) {
+    project.extensions.getByType<BaseExtension>().apply {
```

Comment:
Just note for future that, in theory, this might break if `android` ever stops being `BaseExtension`, but since the original code is affected too and it may happen only with gradle upgrades, it isn't a problem.